proto/v1/idp_service.proto

```diff
@@ -204,7 +221,9 @@ message OIDCIdentityProviderConfig {
 
   string client_secret = 3;
 
-  FieldMapping field_mapping = 4;
+  repeated string scopes = 4;
+
+  FieldMapping field_mapping = 5;
```

Comment:
Is this the right way in gRPC? Or should all new fields only be added to the last?